environments/local-docker.yaml

```diff
@@ -257,4 +257,9 @@ redis:
 redis-commander:
   enabled: true
   service:
-    port: 8181    
\ No newline at end of file
+    port: 8181    
+
+pb-ac-tests:
+  enabled: false
```

Comment:
you might want to start off with it disabled for the azure deployment also

Reply:
Assuming we're concerned about it running in prod, since we will want to run it in dev and test for sure to start with, should we perhaps have it on by default and only disable it in prod rather than the other way round?